docs/index.rst

```diff
@@ -9,8 +9,45 @@ Welcome to Paradise Paper Search App's Django + Neomodel Tutorial!
 
 Overview
 --------
-.. todo::
-   Here will be the tutorial overview.
+
+The aim of this tutorial is to demonstrate how to develop a web application using the Django 
+framework backed by Neo4j, connecting them with the Neomodel driver. Neomodel is an Object Graph 
+Mapper (OGM) for the Neo4j graph database.
+
+The tutorial covers several topics, some of them are: how to configure Neomodel within a Django 
+program, create models, query to the database, create an API. It provides best-practice guidance
+on implementing Neomodel.
+
+The project is a web application to search for information on a Paradise Paper Graph Database. 
+he data from the Database includes companies and people in more than 200 countries that are part 
+of the Paradise Papers, Panama Papers, Bahama Leaks or the Offshore Leaks investigations.
+
+The search of the program can filter the mentioned data by country, jurisdiction and/or data source. 
+The approach adopted to make the program was to create a single-page web application, from which 
```

Comment:
Think 'in which' sounds better than 'from which'. For some reason sounds off to me